tests/test_multi_standbys.py

```diff
@@ -296,3 +296,67 @@ def test_020_start_node3_again():
     assert node1.has_needed_replication_slots()
     assert node2.has_needed_replication_slots()
     assert node3.has_needed_replication_slots()
+
+def test_021_most_advanced_standby_inaccessible():
+
+    global node1
+    global node2
+    global node3
+
+    node1.set_candidate_priority(50)   # the current primary
+    node2.set_candidate_priority(100)  # the next primary
+    node3.set_candidate_priority(0)    # never candidate for a failover
+
+    # to emulate one node is behind, it is easier to make it async
+    # we want node2 to be behind others
+    node1.set_number_sync_standbys(0)
```

Comment:
The comment does not apply to this line. Please comment this line specifically, and move the comment to the next block of three lines.